ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -394,10 +396,12 @@ objects:
           queue /tmp/artifacts/nodes.json oc --insecure-skip-tls-verify --request-timeout=5s get nodes -o json
           queue /tmp/artifacts/openshiftapiserver.json oc --insecure-skip-tls-verify --request-timeout=5s get openshiftapiserver -o json
           queue /tmp/artifacts/pods.json oc --insecure-skip-tls-verify --request-timeout=5s get pods --all-namespaces -o json
+          FILTER=gzip queue /tmp/artifacts/replicasets.json.gz oc --insecure-skip-tls-verify --request-timeout=5s get replicasets --all-namespaces -o json
           queue /tmp/artifacts/rolebindings.json oc --insecure-skip-tls-verify --request-timeout=5s get rolebindings --all-namespaces -o json
           queue /tmp/artifacts/roles.json oc --insecure-skip-tls-verify --request-timeout=5s get roles --all-namespaces -o json
           queue /tmp/artifacts/secrets.json oc --insecure-skip-tls-verify --request-timeout=5s get secrets --all-namespaces -o json
```

Comment:
WTF who added this.  I hope this isn't leaking our GCP keys.

Reply:
It's leaking our AWS keys, please remove the secrets from all jobs and then I'm going to have to purge this from GCP.  Also find out who added this and sternly warn them.

Reply:
it was added by @deads2k 
https://github.com/openshift/release/commit/81a711c1a3680f55b765ad6fbc26f86b1623eaf0#diff-6a0349f9e2bc8f6920d3661f2b81a8e4

@deads2k consider yourself sternly warned by proxy.

and i guess @stevekuznetsov since you approved the pr:
https://github.com/openshift/release/pull/2844

Reply:
@smarterclayton i've updated this PR to remove them.

Reply:
sorry, wrong PR.  it was this one:
https://github.com/openshift/release/pull/2842

still @deads2k but @mtnbikenc lgtm'd it.